app/main.py

```diff
@@ -1,3 +1,14 @@
-def cache(func):
-    # Write your code here
-    pass
+from typing import Callable
+
+
+def cache(func: Callable) -> Callable:
+    list_of_answers = {}
+
+    def calc_or_cache(*args: tuple) -> dict:
```

Comment:
function can return any value
u don't need typehint for *args
```suggestion
    def calc_or_cache(*args) -> Any:
```